cmd/initdb/main.go

```diff
@@ -25,27 +26,58 @@ var entries = []Entry{
 }
 
 const fmtStr = "insert into users (first_name, last_name, email) values ('%s', '%s', '%s')"
+const tableQuery = `SELECT tablename FROM pg_catalog.pg_tables where tablename = $1`
+
+func checkTableExists(db *sql.DB) (bool, error) {
+	rows, err := db.Query(tableQuery, "users")
+	if err != nil {
+		return false, err
+	}
+	defer rows.Close()
+	for rows.Next() {
+		var tableName string
+		err = rows.Scan(&tableName)
+		if err != nil {
+			return false, err
+		}
+		log.Printf("Found existing table: %q\n", tableName)
+		return true, nil
+	}
+	return false, rows.Err()
+}
 
 func main() {
 	db, err := bindingsql.Open(context.TODO(), "postgres")
 	if err != nil {
 		log.Fatal(err)
 	}
 
-	res, err := db.Exec(`CREATE TABLE users (
+	tableExists, err := checkTableExists(db)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	if !tableExists {
+		log.Printf("Creating 'users' table")
+		res, err := db.Exec(`CREATE TABLE users (
                   id SERIAL,
                   first_name VARCHAR(255),
                   last_name VARCHAR(255),
                   email TEXT
                 );`)
-	if err != nil {
-		log.Fatalf("Failed to create table users: %v", err)
+		if err != nil {
+			log.Fatalf("Failed to create table users: %v", err)
+		}
+		log.Printf("Created Table: %+v", res)
 	}
 
-	log.Printf("Created Table: %+v", res)
+	_, err = db.Exec("delete from users")
```

Comment:
nit: capitalize keywords 
```suggestion
	_, err = db.Exec("DELETE FROM users")
```